hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/loghandler/NonAggregatingLogHandler.java

```diff
@@ -209,6 +222,23 @@ public Set<ApplicationId> getInvalidTokenApps() {
     return Collections.emptySet();
   }
 
+  private long calculateSizeOfAppLogs(String user, ApplicationId applicationId) {
+    FileContext lfs = getLocalFileContext(getConfig());
+    long appLogsSize = 0L;
+    for (String rootLogDir : dirsHandler.getLogDirsForCleanup()) {
+      Path logDir = new Path(rootLogDir, applicationId.toString());
+      try {
+        appLogsSize += lfs.getFileStatus(logDir).getLen();
+      } catch (UnsupportedFileSystemException ue) {
+        LOG.warn("Unsupported file system used for log dir " + logDir, ue);
```

Comment:
LOG {}

Reply:
These codes are not added by me, because CheckStyle doesn't pass, so I modified the indentation.